contracts/lib/Approvable.sol

```diff
@@ -3,18 +3,19 @@ pragma solidity ^0.4.11;
 import './Ownable.sol';
 
 contract Approvable is Ownable {
```

Comment:
We should remove `is Ownable` now that we don't use it here. But make sure to update other contracts that inherit from `Approvable` and also use `Ownable`.